src/general/HelpScreen.cs

```diff
@@ -94,7 +94,7 @@ private void BuildHelpTexts(string category)
     {
         var helpTexts = SimulationParameters.Instance.GetHelpTexts(category);
 
-        var middleIndex = helpTexts.Messages.Count / 2;
+        var middleIndex = Math.Ceiling(helpTexts.Messages.Count / 2.0f);
```

Comment:
Referring to previous comment, this switches the ordering which may improve the apparent uneven look of microbe stage's help screen (due to one box being longer than the other), but if you go to the microbe editor it would instead look off balanced because the longer box is on the left side. Either way is fine but I think it's slightly unnecessary to have an extra handling here.

Reply:
Oh yeah that's true. Why is it so hard to get the help texts split evenly? Just to save time, I'd just leave this as-is.

Reply:
Thinking more on this, the old approach of having manually selected left and right side texts may have been the optimal solution...

Reply:
They are alreay more or less even (order-wise) but not so much regarding content length

Reply:
> Thinking more on this, the old approach of having manually selected left and right side texts may have been the optimal solution...

I agree, with the current approach it's more convenient but there's less control on what goes where, maybe we could open an issue to revert to the old approach.

Reply:
:+1: on opening a new issue